freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -96,8 +97,9 @@ public CloudStack convert(Stack stack, Collection<String> deleteRequestedInstanc
         Network network = buildNetwork(stack);
         InstanceAuthentication instanceAuthentication = buildInstanceAuthentication(stack.getStackAuthentication());
 
-        return new CloudStack(instanceGroups, network, image, Collections.emptyMap(), getUserDefinedTags(stack), stack.getTemplate(),
-                instanceAuthentication, instanceAuthentication.getLoginUserName(), instanceAuthentication.getPublicKey(), null);
+        return new CloudStack(instanceGroups, network, image, Map.of(CLOUDWATCH_CREATE_PARAMETER, Boolean.TRUE.toString()),
```

Comment:
this should be also come from a `@Value` so we can turn it off/on as we wish

Reply:
done